### ISSUE TITLE
Can't Access Hidden Files Menu When Running Bottles

### ISSUE THREAD

**Issue by authenticastrology** [pop-os/cosmic-epoch#803]
Having issues with Bottles. I had to go into 22.04 to resolve. But here is the issue for the developers. First, the app won't run unless you unset Wayland. When you do get it to run, when trying to run an exe, Cosmic Files doesn't provide an option to show hidden files and Ctrl+H doesn't work.

**Comment by Department-stack** [comment]
I don't think this is a cosmic issue? I have had these same issues but on kde too.

**Comment by authenticastrology** [comment]
Yes, but it doesn't do it on Pop 22.04. It is easy to access hidden files and so it would be helpful to have the same functionality in Cosmic.

**Comment by michaelBelsanti** [comment]
this should be fixed. the cosmic file picker used to not support showing hidden files, but Ctrl+H shows them now.
